src/Analyzers/VisualBasic/Tests/IncludeParenthesesForMethodInvocations/IncludeParenthesesForMethodInvocationsTests.vb

```diff
@@ -0,0 +1,103 @@
+            Dim test As New VerifyVB.Test With
+            {
+                .TestCode = source,
+                .FixedCode = fixedSource
+            }
+            test.Options.Add(VisualBasicCodeStyleOptions.IncludeParenthesesForMethodInvocations, includeParenthesesForMethod)
+            Await test.RunAsync()
+        End Function
+
+        Private Shared Async Function VerifyNoDiagnosticAsync(source As String, includeParenthesesForMethod As Boolean) As Task
+            Dim test As New VerifyVB.Test With {.TestCode = source}
+            test.Options.Add(VisualBasicCodeStyleOptions.IncludeParenthesesForMethodInvocations, includeParenthesesForMethod)
+            Await test.RunAsync()
+        End Function
+
+#Region "No Diagnostic"
+        <Fact>
+        Public Async Function ParenthesesExistOnInvocation_OptionIsTrue_NoDiagnostic() As Task
+            Dim testCode = "
+Public Module Module1
+    Sub Main()
+        System.Console.WriteLine()
+    End Sub
+End Module
+"
+            Await VerifyNoDiagnosticAsync(testCode, includeParenthesesForMethod:=True)
+        End Function
+
+        <Fact>
+        Public Async Function ParenthesesDoesNotExistOnInvocation_OptionIsFalse_NoDiagnostic() As Task
+            Dim testCode = "
+Public Module Module1
+    Sub Main()
+        System.Console.WriteLine
+    End Sub
+End Module
+"
+            Await VerifyNoDiagnosticAsync(testCode, includeParenthesesForMethod:=False)
+        End Function
+
+#End Region
+
+#Region "Diagnostic"
+
+#End Region
+
+        <Fact>
+        Public Async Function ParenthesesExistOnInvocation_OptionIsFalse_Diagnostic() As Task
+            Dim testCode = "
+Public Module Module1
+    Sub Main()
+        System.Console.WriteLine()
+    End Sub
+End Module
+"
+            '            Dim expected = "
+            'Public Module Module1
+            '    Sub Main()
+            '        System.Console.WriteLine
+            '    End Sub
+            'End Module
+            '"
+            ' This should fail now. This call will be replaced with VerifyCodeFixAsync after the codefix is implemented.
+            Await VerifyNoDiagnosticAsync(testCode, includeParenthesesForMethod:=False)
+        End Function
+
+        <Fact>
+        Public Async Function ParenthesesDoesNotExistOnInvocation_OptionIsTrue_Diagnostic() As Task
+            Dim testCode = "
+Public Module Module1
+    Sub Main()
+        System.Console.WriteLine
+    End Sub
+End Module
+"
+
+            '            Dim expected = "
+            'Public Module Module1
+            '    Sub Main()
+            '        System.Console.WriteLine()
+            '    End Sub
+            'End Module
+            '"
+            ' This should fail now. This call will be replaced with VerifyCodeFixAsync after the codefix is implemented.
+            Await VerifyNoDiagnosticAsync(testCode, includeParenthesesForMethod:=True)
+        End Function
+    End Class
```

Comment:
add fixall test.
add test for a method call that is passed in as an argument to some other method.

Reply:
> add fixall test.

If the test uses the new test library, then _all_ tests are Fix All tests.